assets/src/scripts/monitoring-location/components/hydrograph/discrete-data.js

```diff
@@ -21,17 +17,9 @@ export const drawGroundwaterLevels = function(svg, {levels, xScale, yScale, enab
 
     levels.forEach((level) => {
         group.append('circle')
-            .attr('class', GW_LEVEL_CLASS)
-            .attr('r', GW_LEVEL_RADIUS)
+            .attr('class', level.classes.join(' '))
+            .attr('r', level.radius)
             .attr('cx', xScale(level.dateTime))
             .attr('cy', yScale(level.value));
     });
 };
-
-/*
- * Returns a circle marker that can be used to represent the groundwater level symbol in legends
- * @return {Object} - see d3-rendering/markers module.
- */
-export const getGroundwaterLevelsMarker = function() {
```

Comment:
I hated that this was pulled into selector/legend-data (which I coded originally). By putting all the info in selectors/discrete-data when getting the ground water level points to be able to render the points, this doesn't need to be here (since radius, label and classes are passed with the points.

Reply:
I did think that was an odd way to set the radius/class but it worked under the circumstances.